src/components/search/SearchField.js

```diff
@@ -78,16 +117,25 @@ const SearchField = ({ value, toggle, setToggle, fieldRef, createModal }) => {
                 height: '20vh',
               }}
             >
-              {list.map((e) => (
-                <li
-                  key={e.id}
-                  className="searchBar"
-                  onClick={() => createModal(<StudentInfoModal user={e} />)}
-                >
-                  {' '}
-                  {`${e.first_name} ${e.last_name}`}{' '}
-                </li>
-              ))}
+              {value.name.length > 0 && list.length > 0 ? currentListSections.map((section) => (
+                <>
```

Comment:
gotta get a key on here. easiest way is to make it into a div and use section.title

Reply:
How did I miss this? Easy fix